examples/with-prisma/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
+  "prisma": {
+    "schema": "packages/database/prisma/schema.prisma",
+    "seed": "esno packages/database/src/seed.ts"
```

Comment:
Add esno in devDeps.

Alternatively ts-node with https://github.com/belgattitude/nextjs-monorepo-example/blob/main/packages/db-main-prisma/tsconfig.json

Reply:
Good catch @belgattitude 

I personally prefer the use of `esno` of `ts-node` as it's powered by `esbuild` so it's much faster. Which would you prefer @jaredpalmer ?

Reply:
Hey @NuroDev I switched to esno too :smile:. Nice gem, thx

Reply:
esno is fine